packages/edit-site/src/components/style-book/constants.ts

```diff
@@ -107,6 +107,11 @@ export const STYLE_BOOK_THEME_SUBCATEGORIES: Omit<
 ];
 
 export const STYLE_BOOK_CATEGORIES: StyleBookCategory[] = [
+	{
+		slug: 'landing',
+		title: __( 'Landing' ),
```

Comment:
I know "Landing" has been bandied about, but what does it mean exactly to a user?

From https://github.com/WordPress/gutenberg/issues/53431:

> It serves as a quick glanceable "poster view" for the site style guide

I don't have a better alternative — 'Overview'? — but I suppose it could go in the direction of "Theme style guide" if and when we allow patterns and/or theme devs to add to it.

Reply:
Personally, I think I lean towards "Overview" rather than "Landing" as being more meaningful for more people. 

IIRC there was some talk recently about quirky things and personality being something missed lately. A left field option could be "At a glance". It's not a single word but is still relatively short. Once it's translated that could be a different story though.

Reply:
Great question. I'll just ping @richtabor for discussion on the name for the category as he often has good ideas for these things!

Reply:
Connecting some dots to my wall of [text here](https://github.com/WordPress/gutenberg/pull/66545#issuecomment-2459211973), I expect the tabs to eventually go away in favor of drilldown-based contextual navigation. Instead of clicking a tab, you click "Typography" in global styles, and the preview updates to match.

In that vein, the name of the tab stops becoming user visible.